plugins/Identify.jsx

```diff
@@ -61,7 +66,9 @@ class Identify extends React.Component {
     state = {
         mode: 'Point',
         identifyResults: null,
-        pendingRequests: 0
+        pendingRequests: 0,
+        radius: 50,
+        units: 'meters'
```

Comment:
Small nitpick: units -> radiusUnits, to give implicit context to the state variable.